app/javascript/sdk/IFrameHelper.js

```diff
@@ -192,6 +192,21 @@ export const IFrameHelper = {
       onBubbleClick(bubbleState);
     },
 
+    popoutChatWindow: ({ baseUrl, websiteToken, locale }) => {
+      const cwCookie = Cookies.get('cw_conversation');
+      let widgetUrl = IFrameHelper.getUrl({ baseUrl, websiteToken });
+      if (cwCookie) {
+        widgetUrl = `${widgetUrl}&cw_conversation=${cwCookie}&locale=${locale}`;
+      }
+      const popoutWindow = window.open(
+        widgetUrl,
+        `webwidget_session_${websiteToken}`,
+        'resizable=off,width=400,height=600'
+      );
+      window.$chatwoot.toggle('close');
+      popoutWindow.focus();
```

Comment:
Can't we re-use the method written in widget? It seems like both of these are doing the same. Can we create a shared helper method for it?

Reply:
Moved this to a helper file in https://github.com/chatwoot/chatwoot/pull/4232/commits/f02787c10fea36f7f0d7a9ee846387c6481baf5e